diff --git a/node_modules/@ephox/bedrock-server/lib/main/ts/bedrock/compiler/Webpack.js b/node_modules/@ephox/bedrock-server/lib/main/ts/bedrock/compiler/Webpack.js
index 377cfe7..62a15b9 100644
--- a/node_modules/@ephox/bedrock-server/lib/main/ts/bedrock/compiler/Webpack.js
+++ b/node_modules/@ephox/bedrock-server/lib/main/ts/bedrock/compiler/Webpack.js
@@ -43,7 +43,7 @@ const webpackSharedRules = webpackRemap.concat([
         enforce: 'pre'
     },
     {
-        test: /\.(html|htm|css|bower|hex|rtf|xml|yml)$/,
+        test: /\.(html|htm|css|bower|hex|rtf|xml|yml|svg)$/,
         type: 'asset/source'
     },
     {
